### PR TITLE
i18n(zh-cn): Upgrade to Astro v3 guide for simplified Chinese 

### PR DESCRIPTION
**Pull request by liruifengv**
#### What kind of changes does this PR include?

- New or updated content

#### Description
- Upgrade to Astro v3 guide for simplified Chinese
- uppercase http methods & some update.
- update node version
- translate v3.mdx
- update view-transitions.mdx
- etc.

#### Referecne

https://github.com/withastro/docs/pull/4166

### CODE REVIEW

File: src/content/docs/zh-cn/concepts/why-astro.mdx [hunk 1]
```diff
@@ -56,26 +50,21 @@ Astro 的魔力在于它如何将上述两个值（内容焦点于服务器优
...
-
-
+Astro 的设计比其他UI框架和语言更简单，其中一个重要原因是 Astro 被设计为在服务器上渲染而不是浏览器，这意味着你无需担心：React Hooks、stale closures(React)、refs(Vue)、observables(Svelte)、atoms、selectors、reactions or derivations。服务器上没有响应式，因此这些复杂性都消失了。
 
+我们最喜欢的谚语之一是：**选择加入复杂性。** 我们设计 Astro 是为了尽可能多地从开发人员体验中消除“所需的复杂性”，尤其是你首次加入时。你可以在 Astro中使用 HTML和CSS构建“Hello World”示例网站。然后当你需要构建更强大的功能时，你可以随时获得新功能和 API。
 
 ## 功能齐全且灵活
 
-**Astro 是集多功能于一体的 Web 框架，提供了构建网站所需的一切** Astro 包括组件语法、基于文件的路由、静态资源处理、构建处理、捆绑、优化、数据获取等。您可以在不超过 Astro 核心功能集 的情况下构建出色的网站。
+**Astro 是集多功能于一体的 Web 框架，提供了构建网站所需的一切** Astro 包括组件语法、基于文件的路由、静态资源处理、构建处理、捆绑、优化、数据获取等。你可以在不超过 Astro 核心功能集 的情况下构建出色的网站。
```
Comment (Genteure) [inline src/content/docs/zh-cn/concepts/why-astro.mdx:66]:
```suggestion
**Astro 是集多功能于一体的 Web 框架，提供了构建网站所需的一切** Astro 包括组件语法、基于文件的路由、静态资源处理、构建处理、捆绑、优化、数据获取等。你只使用 Astro 核心功能集就可构建出色的网站。
```

Original English text: `You can build great websites without ever reaching outside of Astro's core feature set.`

Once again while I agree "你可以在不超过 Astro 核心功能集 的情况下构建出色的网站。" is an accurate translation, it's a little too literal, making it hard to understand (to me at least). "Reaching outside" seems hard to translate in this context, so I'm suggesting flipping the sentence from negative to affirmative.

Literal translation back to English: `You can build great websites using only Astro's core feature set.`

File: src/content/docs/zh-cn/concepts/why-astro.mdx [hunk 2]
```diff
@@ -56,26 +50,21 @@ Astro 的魔力在于它如何将上述两个值（内容焦点于服务器优
...
 
-如果您需要更多的控制，你可以通过 [React](https://www.npmjs.com/package/@astrojs/react), [Svelte](https://www.npmjs.com/package/@astrojs/svelte), [Vue](https://www.npmjs.com/package/@astrojs/vue), [Tailwind CSS](https://www.npmjs.com/package/@astrojs/tailwind), [MDX](https://www.npmjs.com/package/@astrojs/mdx), [image optimizations](https://www.npmjs.com/package/@astrojs/image)等[100多个集成](https://astro.build/integrations/)。 扩展 Astro 只需要一个命令 [即可连接您喜欢的CMS](/zh-cn/guides/cms/) 或 [部署到您喜欢的服务器](/zh-cn/guides/deploy/)
+如果你需要更多的控制，你可以通过 [React](https://www.npmjs.com/package/@astrojs/react), [Svelte](https://www.npmjs.com/package/@astrojs/svelte), [Vue](https://www.npmjs.com/package/@astrojs/vue), [Tailwind CSS](https://www.npmjs.com/package/@astrojs/tailwind), [MDX](https://www.npmjs.com/package/@astrojs/mdx) 等[100多个集成](https://astro.build/integrations/)。 扩展 Astro 只需要一个命令 [即可连接你喜欢的 CMS](/zh-cn/guides/cms/) 或 [部署到你喜欢的服务器](/zh-cn/guides/deploy/)
```
Comment (Genteure) [inline src/content/docs/zh-cn/concepts/why-astro.mdx:68]:
```suggestion
如果你需要更多的控制，你可以通过 [React](https://www.npmjs.com/package/@astrojs/react), [Svelte](https://www.npmjs.com/package/@astrojs/svelte), [Vue](https://www.npmjs.com/package/@astrojs/vue), [Tailwind CSS](https://www.npmjs.com/package/@astrojs/tailwind), [MDX](https://www.npmjs.com/package/@astrojs/mdx) 等[100多个集成](https://astro.build/integrations/)。 扩展 Astro 只需要一个命令 [即可连接你喜欢的 CMS](/zh-cn/guides/cms/) 或 [部署到你喜欢的托管平台](/zh-cn/guides/deploy/)
```

Original: `deploy to your favorite host`

While "host" can be translated as "服务器" (literal: server), it seems to me "托管" or "托管平台" is more appropriate.

"服务器" usually means a physical or virtual server that you can SSH into, like AWS Lightsail, EC2, or GCP Compute Engine.
Services that hosts static web resources or provides a serverless runtime certainly does not count as a "server" (it's literally "serverless").

Both AWS and GCP translate "hosting" as "托管".
https://aws.amazon.com/cn/what-is/web-hosting/
https://cloud.google.com/solutions/web-hosting?hl=zh-cn

**Review by Genteure — CHANGES_REQUESTED** [review]
Sorry for nitpicking word choices not made in this PR. 😂

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 3]
```diff
@@ -120,6 +120,41 @@ export default defineConfig({
 目前，`compat` 选项只适用于 React 库，将代码以 ESM 的形式导出。如果在构建时出现错误，请尝试将该库添加到你的`vite.ssr.noExternal: ['the-react-library']` 的 `astro.config.mjs` 文件中。
 :::
 
+## 选项
+
+### 组合对个 JSX 框架
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:125]:
```suggestion
### 组合多个 JSX 框架
```

Typo.

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 4]
```diff
@@ -120,6 +120,41 @@ export default defineConfig({
...
+
+当你在同一个项目中使用多个 JSX 框架（React、Preact、Solid）时，Astro 需要确定每个组件应该使用哪个 JSX 框架特定的转换。如果你只向你的项目中添加了一个 JSX 框架集成，那么就不需要额外的配置。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:127]: Original: `Astro needs to determine which JSX framework-specific transformations should be used for each of your components.`

No suggestion on top of my head yet, but the current translation seems too verbose and literal to me.

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 5]
```diff
@@ -120,6 +120,41 @@ export default defineConfig({
...
+
+使用 `include`（必填）和 `exclude`（可选）配置选项来指定哪些文件属于哪个框架。为你使用的每个框架提供一个文件数组或者文件夹。通配符可用于包含多个文件路径。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:129]:
```suggestion
使用 `include`（必填）和 `exclude`（可选）配置选项来指定哪些文件属于哪个框架。为你使用的每个框架提供一个文件或/和文件夹数组。通配符可用于包含多个文件路径。
```

Original: ```Provide an array of files and/or folders to `include` for each framework you are using.```

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 6]
```diff
@@ -120,6 +120,41 @@ export default defineConfig({
...
+
+我们建议将常见的框架组件放在同一个文件夹中（例如 `/components/react/` 和 `/components/solid/`），以便更容易地指定你的包含内容，但这不是必需的：
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:131]:
```suggestion
我们建议将每个框架的组件放在同一个文件夹中（例如 `/components/react/` 和 `/components/solid/`），以便更容易地指定你的包含内容，但这不是必需的：
```

Original: `We recommend placing common framework components in the same folder...`

My understanding of `common` in this context is `shared by, coming from, or done by more than one`. Currently it's translated as `occurring, found, or done often; prevalent`.

**Review by Genteure — CHANGES_REQUESTED** [review]
These suggestions also apply to other pages containing the same "multiple JSX frameworks" paragraph.

File: src/content/docs/zh-cn/guides/upgrade-to/v3.mdx [hunk 7]
```diff
@@ -0,0 +1,598 @@
...
+
+#### 我应该怎么做？
+
+更新任何现有的引用 `localhost:3000`，例如在测试或在你的 `README` 中，以反映新的端口 `localhost:4321`。
+
+### 默认值改变：import.meta.env.BASE_URL `trailingSlash`
+
+在 Astro v2.x 中，`import.meta.env.BASE_URL` 默认情况下会将你的 [`base`](/zh-cn/reference/configuration-reference/#base) 设置附加到 [trailingSlash](/zh-cn/reference/configuration-reference/#trailingslash)。`trailingSlash: "ignore"` 也会附加一个尾随斜杠。
+
+Astro v3.0 不再默认情况下将 `import.meta.env.BASE_URL` 与尾随斜杠附加在一起，也不会在设置 `trailingSlash: "ignore"` 时附加一个尾随斜杠。（`base` 与 `trailingSlash: "always"` 或 `trailingSlash: "never"` 结合使用的现有行为保持不变。）
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/upgrade-to/v3.mdx:368]: Could use some rewording. Waiting for confirmation of whether or not my understanding of the original English docs is correct before suggesting changes.

**Review by Genteure — CHANGES_REQUESTED** [review]

File: src/content/docs/zh-cn/guides/integrations-guide/react.mdx [hunk 8]
```diff
@@ -111,6 +111,39 @@ export default defineConfig({
 });
 ```
 
+### 组合对个 JSX 框架
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/react.mdx:114]:
```suggestion
### 组合多个 JSX 框架
```

File: src/content/docs/zh-cn/guides/integrations-guide/react.mdx [hunk 9]
```diff
@@ -111,6 +111,39 @@ export default defineConfig({
...
+
+当你在同一个项目中使用多个 JSX 框架（React、Preact、Solid）时，Astro 需要确定每个组件应该使用哪个 JSX 框架特定的转换。如果你只向你的项目中添加了一个 JSX 框架集成，那么就不需要额外的配置。
+
+使用 `include`（必填）和 `exclude`（可选）配置选项来指定哪些文件属于哪个框架。为你使用的每个框架提供一个文件数组或者文件夹。通配符可用于包含多个文件路径。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/react.mdx:118]:
```suggestion
使用 `include`（必填）和 `exclude`（可选）配置选项来指定哪些文件属于哪个框架。为你使用的每个框架提供一个文件或/和文件夹数组。通配符可用于包含多个文件路径。
```

File: src/content/docs/zh-cn/guides/integrations-guide/solid-js.mdx [hunk 10]
```diff
@@ -69,6 +69,41 @@ export default defineConfig({
 *   💧 客户端水合选项
 *   🤝 将框架混合和嵌套在一起的时机
 
+## 选项
+
+### 组合对个 JSX 框架
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/solid-js.mdx:74]:
```suggestion
### 组合多个 JSX 框架
```

File: src/content/docs/zh-cn/guides/integrations-guide/solid-js.mdx [hunk 11]
```diff
@@ -69,6 +69,41 @@ export default defineConfig({
...
+
+当你在同一个项目中使用多个 JSX 框架（React、Preact、Solid）时，Astro 需要确定每个组件应该使用哪个 JSX 框架特定的转换。如果你只向你的项目中添加了一个 JSX 框架集成，那么就不需要额外的配置。
+
+使用 `include`（必填）和 `exclude`（可选）配置选项来指定哪些文件属于哪个框架。为你使用的每个框架提供一个文件数组或者文件夹。通配符可用于包含多个文件路径。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/solid-js.mdx:78]:
```suggestion
使用 `include`（必填）和 `exclude`（可选）配置选项来指定哪些文件属于哪个框架。为你使用的每个框架提供一个文件或/和文件夹数组。通配符可用于包含多个文件路径。
```

**Comment by liruifengv** [comment]
@Genteure Thank you for the detailed review. I fixed them and added some updates from upstream https://github.com/withastro/docs/pull/4166

File: src/content/docs/zh-cn/reference/directives-reference.mdx [hunk 12]
```diff
@@ -26,25 +26,22 @@ title: 模板指令参考
 
 ### `class:list`
 
-`class:list={...}` 接收 class 数组，并将其转换为 class 字符串。这是受流行的 @lukeed [clsx](https://github.com/lukeed/clsx)辅助库的启发，但直接内置在 Astro 里。
+`class:list={...}` 接收 class 数组，并将其转换为 class 字符串。这是受流行的 @lukeed [clsx](https://github.com/lukeed/clsx)辅助库的启发。
```
Comment (Genteure) [inline src/content/docs/zh-cn/reference/directives-reference.mdx:29]:
```suggestion
`class:list={...}` 接收 class 数组，并将其转换为 class 字符串。这是受流行的 @lukeed [clsx](https://github.com/lukeed/clsx) 辅助库的启发。
```

**Review by Genteure — CHANGES_REQUESTED** [review]
Review progress: 41/48 files.

**Comment by liruifengv** [comment]
> Review progress: 41/48 files.

Thanks for your hard work

File: src/content/docs/zh-cn/guides/upgrade-to/v3.mdx [hunk 13]
```diff
@@ -0,0 +1,604 @@
...
+
+在 Astro v2.x 中，[`Astro.cookies.get(key)`](/zh-cn/reference/api-reference/#astrocookies) 总是会返回一个 [`AstroCookie` 对象](/zh-cn/reference/api-reference/#astrocookie)，即使 cookie 不存在。要检查它是否存在，你需要使用 `Astro.cookies.has(key)`。
+
+Astro v3.0 中，如果 cookie 不存在，`Astro.cookies.get(key)` 会返回 `undefined`。
+
+#### 我应该怎么做？
+
+这个更新不会破坏任何在使用 `Astro.cookies.get(key)` 之前检查 `Astro.cookie` 对象是否存在的代码，但现在不再需要。
+
+你可以安全的删除任何使用 `has()` 来检查 `Astro.cookies` 的值是否为 `undefined` 的代码：
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/upgrade-to/v3.mdx:538]:
```suggestion
这个修改不会影响任何在调用 `Astro.cookies.get(key)` 之前用 `Astro.cookies.has(key)` 检查 cookie 是否存在的代码，但现在不再需要检查。

你可以安全的删除任何使用 `has()` 来检查 cookie 是否存在的代码：
```

The suggested change deviate a lot from the original English text. Waiting for feedback on the wording of the original text on Discord.

File: src/content/docs/zh-cn/guides/upgrade-to/v3.mdx [hunk 14]
```diff
@@ -0,0 +1,604 @@
...
+// 构建 Astro 项目
+await build();
+```
+
+
+### 更新：内部 Astro API 入口文件导出路径
+
+在 Astro v2.x 中，你可以从 `astro/internal/*` 和 `astro/runtime/server/*` 导入内部 Astro API。
+
+Astro v3.0 删除了这两个入口文件，而是使用现有的 `astro/runtime/*` 入口文件。此外，还添加了一个新的 `astro/compiler-runtime` 导出，用于编译器特定的运行时代码。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/upgrade-to/v3.mdx:576]:
```suggestion
Astro v3.0 删除了这两个入口文件，而是使用现有的 `astro/runtime/*` 入口文件。此外，还为编译器的运行时代码导出了一个新的 `astro/compiler-runtime`。
```

Preferring natural phrasing over accurate/literal translations.

File: src/content/docs/zh-cn/guides/view-transitions.mdx [hunk 15]
```diff
@@ -1,52 +1,56 @@
...
+Astro 的视图过渡动画支持由新的 [View Transitions](https://developer.chrome.com/docs/web-platform/view-transitions/) 浏览器 API 提供，还包括：
 
-:::caution
-视图过渡动画是 Astro 2.9 中的实验性功能。在标记为稳定之前，API 可能会发生变化。
-:::
+- 一些 [内置的动画选项](#内置动画指令)，例如 `fade`、`slide` 和 `none`。
+- 支持前进和后退导航动画；
+-  可完全 [自定义过渡动画](#自定义动画) 的能力，可以构建自己的动画效果；
+- 用于 [阻止非页面链接的客户端导航](#阻止客户端导航) 的选项。
+-  对于尚未支持视图过渡动画 API 的浏览器，可以 [控制回退行为](#回退控制)。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/view-transitions.mdx:22]:
```suggestion
- 可完全 [自定义过渡动画](#自定义动画) 的能力，可以构建自己的动画效果；
- 用于 [阻止非页面链接的客户端导航](#阻止客户端导航) 的选项。
- 对于尚未支持视图过渡动画 API 的浏览器，可以 [控制回退行为](#回退控制)。
```

Formatting.

File: src/content/docs/zh-cn/guides/view-transitions.mdx [hunk 16]
```diff
@@ -55,27 +59,29 @@ import { ViewTransitions } from 'astro:transitions';
...
 ## 过渡动画指令
 
 Astro 会自动为在旧页面和新页面中找到的相应元素并分配一个共享的唯一`view-transition-name`。这对匹配的元素是通过元素的类型和其在 DOM 中的位置推断出来的。
 
 在你的 `.astro` 组件的页面元素上使用可选的 `transition:*` 指令，以更精确地控制导航期间的页面过渡行为。
 
--  `transition:name`：允许你覆盖 Astro 的默认元素匹配方式，为一对 DOM 元素[指定过渡动画名称](#命名过渡动画)。
+-  `transition:name`：允许你覆盖 Astro 的默认元素匹配方式，为一对 DOM 元素 [指定过渡动画名称](#命名过渡动画)。
 -  `transition:animate`：在用新元素替换旧元素时，允许你通过指定动画类型来覆盖 Astro 的默认动画效果。可以使用 Astro 的 [内置动画指令](#内置的动画指令) 或 [创建自定义动画](#自定义动画)。
 - `transition:persist`: 允许你覆盖 Astro 的默认行为。不同于在导航到另一个页面时用将替换旧元素为新元素的方式，该行为则是在导航到另一个页面时 [保留组件和 HTML 元素的状态](#维持状态)。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/view-transitions.mdx:82]:
```suggestion
- `transition:name`：允许你覆盖 Astro 的默认元素匹配方式，为一对 DOM 元素 [指定过渡动画名称](#命名过渡动画)。
- `transition:animate`：在用新元素替换旧元素时，允许你通过指定动画类型来覆盖 Astro 的默认动画效果。可以使用 Astro 的 [内置动画指令](#内置的动画指令) 或 [创建自定义动画](#自定义动画)。
- `transition:persist`: 允许你覆盖 Astro 的默认行为。不同于在导航到另一个页面时用将替换旧元素为新元素的方式，该行为则是在导航到另一个页面时 [保留组件和 HTML 元素的状态](#维持状态)。
```

Formatting.

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 17]
```diff
@@ -213,12 +213,7 @@ $ astro build --root ./my-project-directory
 
 当使用这个选项时，你所有的静态资源导入和 URL 都需要添加 base 作为前缀。你可以通过 `import.meta.env.BASE_URL` 访问这个值。
 
-默认情况下，`import.meta.env.BASE_URL` 的值包含一个尾部斜杠。如果你将 [`trailingSlash`](/zh-cn/reference/configuration-reference/#trailingslash) 选项设置为 `'never'`，则在静态资源的导入和 URL 中需要手动添加它。
-
-```astro
-<a href="/docs/about/">About</a>
-<img src=`${import.meta.env.BASE_URL}/image.png`>
-```
+`import.meta.env.BASE_URL` 的值会遵从你的 `trailingSlash` 配置，如果你明确地包含一个尾部斜杠，或者设置了 `trailingSlash: "always"`，它将包含一个尾部斜杠。如果设置了 `trailingSlash: "never"`，`BASE_URL` 将不包含尾部斜杠，即使 `base` 中包含。
```
Comment (Genteure) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:216]:
```suggestion
`import.meta.env.BASE_URL` 的值会遵从你的 `trailingSlash` 配置。如果你的 `base` 以斜杠结尾，或者设置了 `trailingSlash: "always"`，它将以斜杠结尾。如果设置了 `trailingSlash: "never"`，即使 `base` 是以斜杠结尾的 `BASE_URL` 也不会以斜杠结尾。
```

Preferring natural phrasing over accurate/literal translations.

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 18]
```diff
@@ -253,19 +248,21 @@ $ astro build --root ./my-project-directory
...
-**默认值：** `'where'`<br />
+**类型：** `'where' | 'class'` | 'attribute'`<br />
+**默认值：** `'attribute'`<br />
 <Since v="2.4" />
 </p>
 
 指定 Astro 组件内部样式作用范围。可选项包括：
   - `'where'` - 使用 `:where` 选择器，不会增加特异性。
   - `'class'` - 使用基于类的选择器，会增加 +1 的特异性。
+  - `'attribute'` 	- 使用 `data-` 属性，不会增加特异性。
```
Comment (Genteure) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:259]:
```suggestion
  - `'attribute'` - 使用 `data-` 属性，不会增加特异性。
```

Formatting.

File: src/content/docs/zh-cn/reference/integrations-reference.mdx [hunk 19]
```diff
@@ -529,6 +535,68 @@ export default defineConfig({
...
+
+Astro 日志记录器的实例，用于写日志。此日志记录器使用与 CLI 配置的相同的[日志级别](/zh-cn/reference/cli-reference/#--verbose)。
+
+用于写入终端的**可用方法**：
+- `logger.info("Message")`;
+- `logger.warn("Message")`;
+- `logger.error("Message")`;
+- `logger.debug("Message")`;
+
+所有消息都前面带有一个具有相同集成值的标签。
```
Comment (Genteure) [inline src/content/docs/zh-cn/reference/integrations-reference.mdx:548]:
```suggestion
所有消息都前面带有一个具有集成名字的标签。
```

Original:

```markdown
All the messages are prepended with a label that has the same value of the integration.
```

File: src/content/docs/zh-cn/reference/integrations-reference.mdx [hunk 20]
```diff
@@ -529,6 +535,68 @@ export default defineConfig({
...
+                const buildLogger = logger.fork("astro-format/build");
+                // do something
+                buildLogger.info("Build finished.")
+            }
+        }
+    }
+}
+```
+
+上面的示例将默认生成带有 `[astro-format]` 的日志，并在指定时生成带有 `[astro-format/build]` 的日志：
```
Comment (Genteure) [inline src/content/docs/zh-cn/reference/integrations-reference.mdx:593]:
```suggestion
上面的示例将默认生成带有 `[astro-format]` 的日志，并在指定名字时生成带有 `[astro-format/build]` 的日志：
```

**Review by Genteure — CHANGES_REQUESTED** [review]
All 48 files reviewed.

File: src/content/docs/zh-cn/guides/upgrade-to/v3.mdx [hunk 21]
```diff
@@ -0,0 +1,604 @@
...
+
+在 Astro v2.x 中，`import.meta.env.BASE_URL` 默认情况下会将你的 [`base`](/zh-cn/reference/configuration-reference/#base) 设置附加到 [trailingSlash](/zh-cn/reference/configuration-reference/#trailingslash)。`trailingSlash: "ignore"` 也会附加一个尾随斜杠。
+
+Astro v3.0 不再默认情况下将 `import.meta.env.BASE_URL` 与尾随斜杠附加在一起，也不会在设置 `trailingSlash: "ignore"` 时附加一个尾随斜杠。（`base` 与 `trailingSlash: "always"` 或 `trailingSlash: "never"` 结合使用的现有行为保持不变。）
+
+#### 我应该怎么做？
+
+如果你的 `base` 已经有了一个尾随斜杠，不需要更改。
+
+如果你的 `base` 没有尾随斜杠，请添加一个，如果你希望保留以前的默认行为（或 `trailingSlash: "ignore"`）。
```
[the remark this answers is not in the document]
Comment (liruifengv) [reply src/content/docs/zh-cn/guides/upgrade-to/v3.mdx:380]: Cool!

**Comment by liruifengv** [comment]
Updated endpoints.mdx & v3.mdx @Genteure

File: src/content/docs/zh-cn/guides/view-transitions.mdx [hunk 22]
```diff
@@ -127,36 +133,42 @@ Astro 会自动为在旧页面和新页面中找到的相应元素并分配一
...
 ```
 
-## 内置的动画指令
+### 内置动画指令
+
+Astro 提供了一些内置的动画选项，用于覆盖默认的 `fade` 过渡动画。在单个元素上添加 `transition:animate` 指令，以自定义特定过渡的行为。
 
-Astro 提供了几种内置动画，用于覆盖默认的 `morph` 过渡效果。可以添加 `transition:animate` 指令来试用 Astro 的 `slide` 或 `fade` 过渡效果。
+- `fade` (默认)：一个基于 Astro 的淡入淡出动画。旧内容淡出，新内容淡入。
+- `initial`: Opt out of Astro's opinionated crossfade animation and use the browser's default styling.
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/view-transitions.mdx:141]:
```suggestion
```

这里忘记删除英语原文了

**Review by 100gle — APPROVED** [review]
LGTM, This is a hard but great work!

**Review by Genteure — APPROVED** [review]
LGTM 👍

**Comment by liruifengv** [comment]
@Genteure @100gle Added some updates from upstream, this may be the last change. (hopefully)

File: src/content/docs/zh-cn/reference/adapter-reference.mdx [hunk 23]
```diff
@@ -187,3 +241,188 @@ if(app.match(request)) {
...
+`entryFile` 是 `URL` 类型，表示文件系统中文件的物理路径。这意味着路径会根据代码运行的操作系统而改变。
+:::
+
+#### 无服务器（Serverless）环境
+
+在无服务器（Serverless）环境中设置 `functionPerRoute: true` 会为每个路由创建一个 JavaScript 文件（handler）。根据你的托管平台，处理程序可能会有不同的名称：lambda、function、page 等。
+
+每个路由都会在处理程序（handler）运行时受到 [冷启动](https://azure.microsoft.com/en-us/blog/understanding-serverless-cold-start/) 的影响，这可能会导致一些延迟。这种延迟受到不同因素的影响。
+
+当设置 `functionPerRoute: false` 时，只有一个单一的处理程序（handler）负责渲染所有路由。当此处理程序（handler）首次触发时，你将受到冷启动的影响。然后，所有其他路由都应该没有延迟。但是，你将失去 `functionPerRoute: true` 提供的代码拆分（code splitting）的好处。
```
Comment (100gle) [inline src/content/docs/zh-cn/reference/adapter-reference.mdx:367]:
```suggestion
当设置 `functionPerRoute: false` 时，只有一个单一的处理程序（handler）负责渲染所有路由。当此处理程序（handler）首次触发时，你将受到冷启动的影响。然后，所有其他路由都应该没有延迟。但是，你将无法享受由 `functionPerRoute: true` 提供的代码拆分（code splitting）所带来的好处。
```

**Review by 100gle — APPROVED** [review]
adapter-reference 的一个部分需要调整一下语序

**Review by Genteure — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
Amazing work everyone, thanks!

**Pull request merged**